### PR TITLE
[CDAP-3253] Feature/ui widget stream dataset

### PR DESCRIPTION
**Pull request by elfenheart**
streamselector

### CODE REVIEW

File: cdap-ui/Gulpfile.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ gulp.task('css:lib', ['fonts'], function() {
       './bower_components/font-awesome/css/font-awesome.min.css',
       './bower_components/ng-sortable/dist/ng-sortable.min.css',
       './bower_components/c3/c3.min.css',
-      './bower_components/angular-gridster/dist/angular-gridster.min.css'
+      './bower_components/angular-gridster/dist/angular-gridster.min.css',
+      './bower_components/angular-ui-select/dist/select.min.css'
```
Comment (ajainarayanan) [inline cdap-ui/Gulpfile.js]: We don't need ui.select anymore right?

**Comment by ajainarayanan** [comment]
@elfenheart Do we need references of ui.select directive? We don't use it anymore right? Is it possible to remove them?

**Comment by elfenheart** [comment]
@ajainarayanan I'll remove them

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**